build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: bgppolicies.crd.antrea.io
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha2
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - nodeSelector
+                - localASN
+                - advertisements
+                - bgpPeers
+              properties:
+                nodeSelector:
+                  type: object
+                  properties:
+                    matchExpressions:
+                      items:
+                        properties:
+                          key:
+                            type: string
+                          operator:
+                            enum:
+                              - In
+                              - NotIn
+                              - Exists
+                              - DoesNotExist
+                            type: string
+                          values:
+                            items:
+                              type: string
+                              pattern: "^(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?$"
+                            type: array
+                        type: object
+                      type: array
+                    matchLabels:
+                      x-kubernetes-preserve-unknown-fields: true
+                localASN:
+                  type: integer
+                  minimum: 64512
+                  maximum: 65535
+                  format: int32
+                listenPort:
+                  type: integer
+                  minimum: 1
+                  maximum: 65535
+                  format: int32
+                  default: 179
+                advertisements:
+                  type: object
+                  anyOf:
+                    - required: [service]
+                    - required: [pod]
+                    - required: [egress]
```

Comment:
Does that mean that at least one of `service`, `pod` and `egress` must be set? If yes, I think we should indicate it as a comment for the `Advertisements` struct in types.go.
Don't we want to allow "empty" advertisements by the way (nothing advertised)?

Reply:
Allowing "empty" advertisements makes sense to me. I'll update this.